docs/data retrieval guide/DataRetrievalGuide.md

```diff
@@ -0,0 +1,173 @@
+
+Table of Contents
+- [ Catena-X Data Retrieval Guide](#-catena-x-data-retrieval-guide)
+  - [Table of Contents](#table-of-contents)
+  - [Starting Point](#starting-point)
+    - [Available Central Services](#available-central-services)
+  - [Problems Generated](#problems-generated)
+  - [Data Retrieval Flow](#data-retrieval-flow)
+    - [1. Discovery Phase](#1-discovery-phase)
+    - [2. Digital Twin Registry Search Phase](#2-digital-twin-registry-search-phase)
+    - [3. Digital Twin Search Phase](#3-digital-twin-search-phase)
+    - [4. Data Negotiation and Transfer Phase](#4-data-negotiation-and-transfer-phase)
+  - [1. Discovery Phase + 2. Digital Twin Registry Search Phase](#1-discovery-phase--2-digital-twin-registry-search-phase)
+    - [Prerequisites](#prerequisites)
+    - [Sequence Diagram](#sequence-diagram)
+    - [Flow Diagram](#flow-diagram)
+  - [3. Digital Twin Search Phase](#3-digital-twin-search-phase-1)
+    - [Prerequisites](#prerequisites-1)
+    - [Sequence Diagram](#sequence-diagram-1)
+    - [Flow Diagram](#flow-diagram-1)
+  - [4. Data Negotiation and Transfer Phase](#4-data-negotiation-and-transfer-phase-1)
+    - [Prerequisites](#prerequisites-2)
+    - [Sequence Diagram](#sequence-diagram-2)
+    - [Flow Diagram](#flow-diagram-2)
+      - [Negotiation and Transfer](#negotiation-and-transfer)
+      - [Data Retrieval](#data-retrieval)
+  - [NOTICE](#notice)
+
+## Starting Point
+
+Before we start with the diagrams is necessary to give some context and remark how things are happening and which services are available.
+
+After the Digital Twin Registry became an decentral component provided by several Catena-X Members as Providers, there was the need of providing central services to enable searching for this digital twin registries.
+
+Therefore some central services were created, allowing the authorized Catena-X Applications to find the EDC endpoints after calling the following services:
+
+### Available Central Services
+
+
+| Service Name | Description | Reference Implementation |
+|------------- | ----------- | ------------------------ |
+| Discovery Service | Responsible to give the search endpoints for a type of id | [eclipse-tractusx/sldt-discovery-finder](https://github.com/eclipse-tractusx/sldt-discovery-finder) |
+| BPN Discovery	| Responsible for indicating the BPNs for the IDs registered by the providers | [eclipse-tractusx/sldt-bpn-discovery](https://github.com/eclipse-tractusx/sldt-bpn-discovery) |
+| EDC Discovery	| Responsible for giving the EDC endpoints of one or more BPNs | [eclipse-tractusx/portal-backend](https://github.com/eclipse-tractusx/portal-backend) - [Code Implementation](https://github.com/eclipse-tractusx/portal-backend/blob/aca855c857aed309cbca03f4f694283629197110/src/administration/Administration.Service/Controllers/ConnectorsController.cs#L178C1-L190C63) |
```

Comment:
This seems to be very specific. Is there a hint, which Portal version the implementation is part of? What value does it provide to point out implementation details, without a version(-range) it is included?